app/lib/appenv.ts

```diff
@@ -4,6 +4,10 @@ export function isProductionMode(): boolean {
 	return import.meta.env.MODE === "production";
 }
 
+export function isMainNetNetwork(): boolean {
+	return import.meta.env.NETWORK === "mainnet";
+}
```

Comment:
this is wrong, this should be based on the wallet information.

Reply:
Done